src/backends/tensorrt/plan_backend.cc

```diff
@@ -228,21 +251,53 @@ PlanBackend::CreateExecutionContext(
   RETURN_IF_ERROR(
       LoadPlan(mn_itr->second, &context->runtime_, &context->engine_));
 
-  if (context->max_batch_size_ > context->engine_->getMaxBatchSize()) {
+  /* Batch Size will be specified explicitly in dimensions
```

Comment:
We still need the concept of max-batch-size from the model configuration to enable the dynamic batcher. Doesn't getMaxBatchSize still work for the engine for non-variable-size models?